README.md

```diff
@@ -44,10 +46,10 @@ The repository contains the infrastructure for both a shared library (librubypar
 
 To compile the shared library, you will need:
 
-* A C99 compiler
-* autotools (autoconf, automake, libtool)
-* make
-* Ruby 3.3.0-preview1 or later
+- A C99 compiler
+- autotools (autoconf, automake, libtool)
+- make
+- Ruby 3.3.0-preview1 or later
```

Comment:
I actually hadn't realized I'd made these changes; I have my IDE setup to run `prettier` on save. Happy to revert if anyone cares.

Reply:
Yes, no unneeded changes please, it's already big enough of a PR

Reply:
Reverted